### PR TITLE
Added risk assessment to contributing page

### PR DESCRIPTION
**Pull request by gabrieldagama**
## Purpose of this pull request

This pull request (PR) adds a description of the pull request risk assessment and risk label descriptions.

## Affected DevDocs pages

-  https://devdocs.magento.com/contributor-guide/contributing.html

whatsnew
Added the ' Pull request risk assessment' section to the [Contributing](https://devdocs.magento.com/contributor-guide/contributing.html) topic.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/contributor-guide/contributing.md [hunk 1]
```diff
@@ -95,6 +95,16 @@ The following list consists of questions you can ask to help determine the prope
 
 The number of 'Yes' answers should help you to determine the severity.
 
+### Pull request risk assessment:
+
+The risk label highlights the risk that the suggested changes may bring to the platform, it helps us to process pull requests, to decide on which version the pull requests should be delivered, it highlights pull requests that may need to be reviewed by other people, and it helps to decide whether a pull request should be approved or not depending on its risk.
+
+| Risk | Description |
+| ------------- | ------------- |
+| High | A pull request that makes changes on the framework or changes that will affect other areas. |
```
Comment (BarnyShergold) [inline src/contributor-guide/contributing.md:104]:
```suggestion
| High | A pull request that makes changes on the framework or changes that will affect multiple areas. |
```

File: src/contributor-guide/contributing.md [hunk 2]
```diff
@@ -95,6 +95,16 @@ The following list consists of questions you can ask to help determine the prope
...
+| Medium | A pull request that makes changes that may affect other areas or it makes considerable changes on specific area. |
```
Comment (BarnyShergold) [inline src/contributor-guide/contributing.md:105]:
```suggestion
| Medium | A pull request that makes changes which may affect multiple areas or makes considerable changes on a specific area. |
```

File: src/contributor-guide/contributing.md [hunk 3]
```diff
@@ -95,6 +95,16 @@ The following list consists of questions you can ask to help determine the prope
...
+| Low | A pull request that it's changes likely won't affect other areas. |
```
Comment (BarnyShergold) [inline src/contributor-guide/contributing.md:106]: Should avoid contractions in technical documents - people with English as a second language may not understand.
```suggestion
| Low | A pull request that will probably not affect other areas. |
```

File: src/contributor-guide/contributing.md [hunk 4]
```diff
@@ -95,6 +95,16 @@ The following list consists of questions you can ask to help determine the prope
 
 The number of 'Yes' answers should help you to determine the severity.
 
+### Pull request risk assessment:
+
+The risk label highlights the risk that the suggested changes may bring to the platform, it helps us to process pull requests, to decide on which version the pull requests should be delivered, it highlights pull requests that may need to be reviewed by other people, and it helps to decide whether a pull request should be approved or not depending on its risk.
```
Comment (BarnyShergold) [inline src/contributor-guide/contributing.md:100]: Sentence is too long - whole paragraph is one sentence
```suggestion
The risk label highlights the risk that the suggested changes may bring to the platform. It helps the Document Maintainer Team to decide on which version the pull requests should be delivered, which may need to be reviewed by other people and whether a request should be approved or not depending on its risk.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Great addition to the docs by the way!

File: src/contributor-guide/contributing.md (hunk 4 above)
Comment (gabrieldagama) [reply src/contributor-guide/contributing.md]: Hi @BarnyShergold, thanks for reviewing, I've partially applied the suggestion, I've replaced `Document Maintainer Team` with `maintainers` which seems to be more appropriate in this case.

**Review by BarnyShergold — APPROVED** [review]
Perfect!

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Pull request merged**